operator/watchers/pod.go

```diff
@@ -4,6 +4,7 @@
 package watchers
 
 import (
+	operatorOption "github.com/cilium/cilium/operator/option"
```

Comment:
Please move this line down next to the other `github.com/cilium/cilium` imports.